votor/src/consensus_pool/stats.rs

```diff
@@ -46,6 +45,40 @@ impl CertificateStats {
     }
 }
 
+#[derive(Default)]
+struct VoteStats {
+    notarize: u64,
+    finalize: u64,
+    skip: u64,
+    notarize_fallback: u64,
+    skip_fallback: u64,
+}
+
+impl VoteStats {
+    fn record(&mut self, vote: &Vote) {
+        match vote {
+            Vote::Notarize(_) => self.notarize = self.notarize.saturating_add(1),
+            Vote::NotarizeFallback(_) => {
+                self.notarize_fallback = self.notarize_fallback.saturating_add(1)
+            }
+            Vote::Skip(_) => self.skip = self.skip.saturating_add(1),
+            Vote::SkipFallback(_) => self.skip_fallback = self.skip_fallback.saturating_add(1),
+            Vote::Finalize(_) => self.finalize = self.finalize.saturating_add(1),
+        }
+    }
+
+    fn submit(&self) {
```

Comment:
nothing wrong with submit per se, but report* would match existing convention better

Reply:
Should be addressed in https://github.com/anza-xyz/agave/pull/8751.